### PR TITLE
extra-features to dev

### PR DESCRIPTION
**Pull request by angarita-dev**

### CODE REVIEW

File: src/js/script.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  let title = document.querySelector("#title").value;
+  let author = document.querySelector("#author").value;
+  let numPages = document.querySelector("#numPages").value;
+  let read = document.querySelector("#read").value;
+
+  let newBook = new Book(title, author, numPages, read);
+  myLibrary.push(newBook);
+  render();
+  document.getElementsByClassName("input")[0].style.display = "none"; 
+  alert(newBook.title + " was added successfully !!!");
```
Comment (JamesOkunlade) [inline src/js/script.js]: The use of `alert` is a bad practice. You should display a `div` above the form if you want to display notification.

File: src/js/script.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+function addBookToLibrary() {
+  let title = document.querySelector("#title").value;
+  let author = document.querySelector("#author").value;
+  let numPages = document.querySelector("#numPages").value;
+  let read = document.querySelector("#read").value;
+
+  let newBook = new Book(title, author, numPages, read);
+  myLibrary.push(newBook);
+  render();
+  document.getElementsByClassName("input")[0].style.display = "none";
```
Comment (JamesOkunlade) [inline src/js/script.js]: The `addBookToLibrary` logic should be free of any `DOM` related code. Refactor your code to ensure all `DOM` tasks are done in the `render` function.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/stylesheet.css">
+  <title>Library App</title>
+</head>
+<body>
+  <div class="button-container">
+    <button id="newBook" onclick="newBook()">New Book</button>
+    <button onclick="render()">Show books</button>
```
Comment (JamesOkunlade) [inline src/index.html]: I don't see need for this `button` since the books display already.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hi, dev! ✋ Good job using the `Git flow` 👏 😄 Your projects looks really good too. See my comments on your code to see minor changes you'll have to make.

- You can decide to persist these books into the `localstorage` as well so the books don't disappear upon refresh. **`[Optional]`**.

**Fix the points I mentioned in the comments and request one last review.**

**Status: `Changes required`**

Happy coding! 😎 


**P.S**

**May I know why you're working on this project alone?**

File: src/index.html (hunk 3 above)
Comment (ngodi) [reply src/index.html]: removed

**Review by BalintMendli — CHANGES_REQUESTED** [review]
Hi @codingAngarita @ngodi ,

Great job on the project. 👏 

However, the linter is not set up correctly. Please add the [javascript config](https://github.com/microverseinc/linters-config/tree/master/javascript) for Stickler and fix lint errors in the code.

When done please submit another review request.

**Comment by angarita-dev** [comment]
# Regarding Stickler error
Stickler throws 4 errors `is defined but never used. (no-unused-vars)`. This is caused because those functions are being called from `index.html`

# Regarding 'working alone'
I did this project along @ngodi we just had to re-upload it to GitHub because we had problems with `git-flow`

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hi dev!! The code is failing some linter checks. Refactor your code to ensure all checks are being passed, It is important to get a pass on this project. :sunglasses

**Request for another review when you're done this.**

**Status: `Changes required`**

**Comment by angarita-dev** [comment]
# Hi @JamesOkunlade
On my previous comment, I stated that the linter didn't detect that the function was being called from the `index.html` therefore the only way I can fix this "error" is to add the rule on the linter config to ignore the `no-unused-vars` rule but I don't think that is a good practice, how may this be addressed?.

> # Regarding Stickler error
> 
> Stickler throws 4 errors `is defined but never used. (no-unused-vars)`. This is caused because those functions are being called from `index.html`

**Comment by AdnanAfsari** [comment]
Hello @codingAngarita @ngodi 👏 
Nice work guys, Sometime its ok to write rule but too much of them are  bad,Keep it like this for the moment .Its a 👍 from me.

**Status: Approved**

Happy Coding

**Pull request merged**